public/scripts/main.js

```diff
@@ -9,6 +11,7 @@ function preload() {
 
 function setup() {
 	createCanvas(1200, 1000);
+	frameRate(20)
```

Comment:
y tho?

Reply:
It got a bit cpu intensive for my computer so I had to lower it. I will adjust it back